share/system/cgroup_linux.go

```diff
@@ -595,6 +627,9 @@ func (s *SystemTools) getMemoryStats(path string, mStats *CgroupMemoryStats, bFu
 
 // https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/fs2/cpu.go
 func (s *SystemTools) statCpu(path string, stats *CpuStats) error {
+	if path == "" {
```

Comment:
Keeping this and getMemoryStats() for the other functions that call this but aren't aware that the path can be empty.